### PR TITLE
 Adds styles and behaviour changes in response to user discussions

### PR DESCRIPTION
**Pull request by schalkventer**
screencapture-localhost-dataset-new-2018-06-24-20_16_34

screencapture-localhost-dataset-new_resource-safhdsdj-2018-06-24-20_16_44

screencapture-localhost-dataset-new-2018-06-24-20_30_58

screencapture-localhost-dataset-new_resource-sfahdsjdf-2018-06-24-20_31_20

### CODE REVIEW

**Comment by schalkventer** [comment]
Note that issues with hamburger icon and placement of stages bar is inherited from current production code:

image

**Comment by longhotsummer** [comment]
"Back to..." instead of "Return"

File: ckanext/satreasury/templates/footer.html [hunk 1]
```diff
@@ -1,28 +1,28 @@
 {% ckan_extends %}
 
 {% block footer_links %}
-  <li>This datastore powers the <a href="https://vulekamali.gov.za">vulekamali.gov.za</a> website.</li>
+  <li>This datastore powers the <a target="_blank" href="https://vulekamali.gov.za">vulekamali.gov.za</a> website.</li>
   <li>&nbsp;</li>
-  <li><a href="https://vulekamali.gov.za/">vulekamali Budget Portal</a></li>
-  <li><a href="https://vulekamali.gov.za/2017-18/about">About vulekamali</a></li>
+  <li><a target="_blank" href="https://vulekamali.gov.za/">vulekamali Budget Portal</a></li>
+  <li><a target="_blank" href="https://vulekamali.gov.za/2017-18/about">About vulekamali</a></li>
```
Comment (longhotsummer) [inline ckanext/satreasury/templates/footer.html]: https://vulekamali.gov.za/about

File: ckanext/satreasury/templates/page.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{% ckan_extends %}
+
+{% block header %}
+  <div class="utility-bar">
+    <div class="utility-bar-center">
+      <a class="utility-bar-link" href="https://vulekamali.gov.za/">← Return to vulekamali</a>
```
Comment (longhotsummer) [inline ckanext/satreasury/templates/page.html]: Return -> Back

**Pull request merged**